frontend/src/entities/dataset/dataset-drag-and-drop.tsx

```diff
@@ -0,0 +1,152 @@
+/**
+  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ 
+  Licensed under the Apache License, Version 2.0 (the "License").
+  You may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+ 
+      http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import React from 'react';
+import { useEffect } from 'react';
+import { useAppDispatch } from '../../config/store';
+import NotificationService from '../../shared/layout/notification/notification.service';
+import { fileHandler } from './dataset.actions';
+
+
+export const FullScreenDragAndDrop = ({state, setState, updateDatasetContext, setDisableUpload}): React.ReactNode => {
+    const dispatch = useAppDispatch();
+    const notificationService = NotificationService(dispatch);
+    const {manageMode} = state;
+
+    async function handleDrop (event) {
+        // Prevent default behavior (Prevent file from being opened)
+        event.preventDefault();
+        document.getElementsByClassName('dropzone')[0].classList.add('display-none');
+        fileHandler(await getFilesAsync(event.dataTransfer), manageMode, notificationService, {state, setState, updateDatasetContext, setDisableUpload});
+    }
+
+    function dragEnterHandler (event) {
+        document.getElementsByClassName('dropzone')[0].classList.add('drag-over');
+        if (event.target.classList.contains('dropzone')) {
+            event.target.classList.add('drag-over');
+        }
```

Comment:
If we're only applying the event handler to `div.dropzone` what scenario is this if statement not true?

Reply:
This should be removed. This is old code. Thanks for catching it!